src/Microsoft.Azure.WebJobs.Logging/FunctionInstanceLogItem.cs

```diff
@@ -29,9 +28,13 @@ public class FunctionInstanceLogItem : IFunctionInstanceBaseEntry
         /// <summary>Gets or sets the Function ID of the ancestor function instance.</summary>
         public Guid? ParentId { get; set; }
 
-        /// <summary>Name of this function. This can be used in querying other instances of this function. </summary>
+        /// <summary>Short Name of this function.</summary>
         public string FunctionName { get; set; }
```

Comment:
Doesn't FunctionId already contain the FunctionName as a component? Can it expose it?